test/topojson/test.ml

```diff
@@ -105,13 +105,60 @@ let get_foreign_members_in_point (f : Topojson.Topology.t) =
   | _ -> assert false
 
 let pp_ezjsonm ppf json = Fmt.pf ppf "%s" (Ezjsonm.value_to_string json)
+let tt = Topojson.Geometry.geometry
```

Comment:
Are these three lines needed anymore ?

Reply:
line 107 is needed in `pp_foreign_member`.
 Let me remove the other two lines

Reply:
Sorry the viewer on Github is not showing what I wanted, I meant lines 108, 109, 110